src/QMCWaveFunctions/tests/CMakeLists.txt

```diff
@@ -47,7 +47,7 @@ IF ((NOT QMC_COMPLEX) AND (NOT QMC_MIXED_PRECISION) AND (NOT QMC_CUDA))
 ENDIF()
 
 ADD_EXECUTABLE(${UTEST_EXE} test_wf.cpp test_bspline_jastrow.cpp test_einset.cpp test_pw.cpp
-               test_polynomial_eeI_jastrow.cpp test_dirac_det.cpp test_dirac_matrix.cpp
+               test_polynomial_eeI_jastrow.cpp test_dirac_det.cpp test_multi_dirac_determinant.cpp test_dirac_matrix.cpp
```

Comment:
Did you forget to include test_multi_dirac_determinant.cpp?

Reply:
I could not compile without removing that test.

Reply:
@PDoakORNL if adding the missing file is the last change of this PR, just add the file. If more changes will be needed, it is better to remove the test here and create a separate PR with the test and following changes. Your call.